client/src/components/DeveloperProfile/DeveloperProfile.jsx

```diff
@@ -0,0 +1,24 @@
+import "./DeveloperProfile.css";
+
+const DeveloperProfile = ({ person }) => {
+  return (
+    <div className="person-card">
+      <h2 className="person-name"> Name: {person.name}</h2>
+      <p className="person-role"> Role: {person.role}</p>
+      <p className="person-github">
+        GitHub:{" "}
+        <a href={`https://github.com/${person.githubUsername}`}>
+          {person.githubUsername}
+        </a>
+      </p>
+      <p className="person-linkedin">
+        LinkedIn:{" "}
+        <a href={`https://www.linkedin.com/in/${person.linkedinUsername}`}>
```

Comment:
I am not sure this would work by just adding the linkedin username because some of our profile links have numbers in them. Have you noticed that? How do we resolve this? Maybe hardcoding the profiles?